web/yo/app/scripts/controllers/nav.js

```diff
@@ -23,12 +23,13 @@ angular.module('oncokbApp')
             filterTabs.push('genes');
             filterTabs.push('queues');
             filterTabs.push('therapies');
-            if ($rootScope.me.admin) {
+            filterTabs.push('tools');
+            if ($rootScope.me && $rootScope.me.admin) {
                 filterTabs = _.union(filterTabs, ['variant', 'tools', 'feedback']);
             }
 
             if (!$rootScope.internal) {
-                filterTabs = _.intersection(filterTabs, ['genes', 'queues', 'feedback']);
+                filterTabs = _.intersection(filterTabs, ['genes', 'queues', 'feedback', 'therapies', 'tools']);
```

Comment:
Does it mean we can only see the `Variant Annotation` tab when we are in external network?

Reply:
This is intersection when user is in external. Variant Annotation should only available internal